### DISCUSSION TITLE
PocketHost is rebooting

### DISCUSSION THREAD

**Discussion by yulei-chen** [pockethost/pockethost#359] [Q&A] [answered]
Thanks for this project. But I hit problem, could anybody help me
When I enter my admin, I always get
![截屏2024-01-16 17 55 22](https://github.com/pockethost/pockethost/assets/27288153/2d473d52-bddc-4ff4-9877-8da0b809167e)

The log looks like:
![截屏2024-01-16 17 59 15](https://github.com/pockethost/pockethost/assets/27288153/f7c4562d-248a-40a2-b5c3-7212baf9f18a)

**Comment by benallfree** [comment]
Hi, thanks for this report. Can you please tell me your instance name so I can take a look?

You can also DM me and chat on Discord instead https://discord.gg/nVTxCMEcGT

**Comment by benallfree** [comment, marked as answer]
Hello, this turned out to be an internal problem affecting your instance and 3 others. Thanks for the report, I fixed it for everyone :)
